grails-app/views/occurrence/list.gsp

```diff
@@ -258,7 +255,7 @@
                                 <%-- Party like it's 1990 --%>
                                 <input type="hidden" name="sort" value="${sort}" />
                                 <input type="hidden" name="dir" value="${dir}" />
-                                <input type="hidden" name="pageSize" value="${pageSize}" />
+                                <input type="hidden" name="pageSize" value="${params.pageSize}" />
```

Comment:
You sure about that? IIRC, ``pageSize`` was here because it's always available, while ``params.pageSize`` only works with queryparams.